ISSUE_TEMPLATE.md

```diff
@@ -1 +1,47 @@
+<!--
+Please use the below format to report the issue in a better way.
+To check any option, replace the space between the "[" and "]" with a "x".
+-->
 
+#### Nature of Issue ?
+
+- [ ] Found a Bug
+- [ ] Existing Feature Enhancement
+- [ ] New Feature Request
+
+#### Most Appropriate Sub Area ?
+
+- [ ] Color
+- [ ] Core
+- [ ] Events
+- [ ] Image
+- [ ] IO
+- [ ] Math
+- [ ] Typography
+- [ ] Utilities
+- [ ] WebGL
```

Comment:
I suggest adding `Mobile` and `Others` here. `Mobile` because it can help speed up isolation of problem (touch based or mobile sensor based etc), `Others` is just a catch all in case something really weird comes along.